transport/http/handler.go

```diff
@@ -144,8 +143,12 @@ func handleOnewayRequest(
 	}
 	treq.Body = &buff
 
+	// create a new context for oneway requests since the HTTP handler cancels
+	// http.Request's context when ServeHTTP returns
+	ctx := opentracing.ContextWithSpan(context.Background(), span)
```

Comment:
Yeah, I agree HTTP is not the best fit for oneway. However, I think we should keep (even if only experimental) since it's relatively 'cheap' way of testing oneway flow/plumbing.

The case of a context deadline exceeding should only cancel the request if it takes too long to 'enqueue' an item. 

Delivery for HTTP simply means that the server has received the entire request and is prepared to process it with no more client interaction.